scripts/cnode-helper-scripts/blockPerf.sh

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2086,SC2281
+
+# Script to collect block information across nodes to provide comprehensive analytics data from participants
+# For now, the script is intended for mainnet network only.
+
+#Todo:
+# - adapt for node names != cnode
+
+# Outside the cnTools environment you can manually point this script to your nodes config.json file
+CONFIG=""  # for example "/home/user/mynode/logs/node0.json"
```

Comment:
Setting it to "" will overwrite any environment assigned vars, prolly good to use sample format as in [cnode.sh](https://github.com/cardano-community/guild-operators/blob/alpha/scripts/cnode-helper-scripts/cnode.sh#L12-L15) that has user variables commented out...and in absence of any defined variables, sets defaults as [here](https://github.com/cardano-community/guild-operators/blob/alpha/scripts/cnode-helper-scripts/cnode.sh#L37-L46).

Reply:
done